config/rubocop.yml

```diff
@@ -92,3 +92,8 @@ AllCops:
 Style/BlockDelimiters:
   Exclude:
     - "spec/**/*"
+
+# Ruby version
+
+AllCops:
+  TargetRubyVersion: 2.3
```

Comment:
not sure, not even sure now if it doesn't take .ruby-version if it exists by default, so yours might still work with .ruby-version set to something different